lib/components/sidebar/sidebar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Drawer as AntDrawer } from 'antd';
+import Container from './container';
+import styles from './sidebar.css';
+
+const Sidebar = React.memo(({ visibleDrawer, cardsData, drawerProps }) => {
```

Comment:
Instead of `cardsData` can it be `children`? Then instead of passing objects we can pass React components.

Reply:
fixed